arrus/core/devices/us4r/Us4RImpl.cpp

```diff
@@ -94,10 +94,7 @@ std::vector<Us4RImpl::VoltageLogbook> Us4RImpl::logVoltages(bool isHV256) {
         voltage = this->getMeasuredMVoltage();
         voltages.push_back(VoltageLogbook{std::string("HVM on HV supply"), voltage, VoltageLogbook::Polarity::MINUS});
     }
-
-    //Verify measured voltages on OEMs
-    auto isUs4OEMPlus = this->isUs4OEMPlus();
-    if(!isUs4OEMPlus || (isUs4OEMPlus && !isHV256)) {
+    else if(isHVPS) {
```

Comment:
I think that change also removes UCD voltage verification on the legacy OEMs, am I correct?